### PR TITLE
feat: style and config sorting filters panel

### PR DESCRIPTION
**Pull request by diegopf**
EX-6345

### CODE REVIEW

**Comment by tiborux** [comment]
Yous miss the divider between facets:

image

File: src/components/search/sort.vue [hunk 1]
```diff
@@ -1,41 +1,46 @@
...
-  >
-    <template #toggle="{ item }">
-      <span>{{ $t('sort.label') }}</span>
-      <!--TODO: remove the `x-text` class when this task done:
-      https://searchbroker.atlassian.net/browse/EX-4196-->
-      <span class="x-text x-dropdown__toggle-selected-value x-list__item--expand">
-        {{ item || 'Default' }}
-      </span>
-      <ChevronTinyDownIcon />
+  <CustomHeaderTogglePanel :data-test="'sort'">
```
Comment (tiborux) [inline src/components/search/sort.vue:2]: You have to add here a gap of 16px:

image

File: src/components/mobile/mobile-close-aside.vue [hunk 2]
```diff
@@ -1,16 +1,43 @@
...
+      v-slot="{ selectedFilters }"
+      class="
+        x-button--secondary x-button--round
+        x-padding--05 x-padding--top-04 x-padding--bottom-04
+        x-uppercase
+        x-list__item--expand
+      "
+      :alwaysVisible="false"
+    >
+      {{ $t('selectedFilters.clear', { selectedFiltersNumber: selectedFilters.length }) }}
```
Comment (tiborux) [inline src/components/mobile/mobile-close-aside.vue:19]: Remove the white spaces in the translations:

`"clear": "Clear {selectedFiltersNumber} filters"`

**Review by tiborux — CHANGES_REQUESTED** [review]
gj man

File: src/i18n/messages/es.messages.json [hunk 3]
```diff
@@ -48,7 +52,7 @@
     },
     "toggleAside": {
       "hideAside": "Ocultar Filtros",
-      "showAside": "Ordenar y Filtrar"
+      "showAside": "Ordenar & filtrar"
```
Comment (tiborux) [inline src/i18n/messages/es.messages.json:55]:
```suggestion
      "showAside": "Ordenar y filtrar"
```

File: src/i18n/messages/es.messages.json [hunk 4]
```diff
@@ -48,7 +52,7 @@
     },
     "toggleAside": {
       "hideAside": "Ocultar Filtros",
```
Comment (tiborux) [inline src/i18n/messages/es.messages.json:54]: remove this message is not longer neccesary.

**Comment by ajperezbau** [comment]
image

The line in the mobile panel should be full width.

File: src/components/desktop/desktop-aside.vue [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+      </span>
+      <BaseIdModalClose class="x-button--ghost x-button--ghost-end" modalId="right-aside">
+        <CrossTinyIcon class="x-icon--l x-font-color--neutral-10" />
+      </BaseIdModalClose>
+    </div>
+    <div class="x-scroll x-list__item--expand x-padding--08">
+      <Sort />
+      <CustomFacets
+        v-if="$x.totalResults > 0"
+        class="x-list--gap-06 x-list--padding-05 x-list--padding-top"
```
Comment (ajperezbau) [inline src/components/desktop/desktop-aside.vue:22]: These classes can be deleted if the facets headers spacings are set using the padding token.

File: src/components/mobile/mobile-aside.vue [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+  <div
+    v-if="$x.totalResults > 0"
+    class="
+      x-list x-list--vertical x-list__item--expand
+      x-padding--05
+      x-list--padding-left x-list--padding-right
+    "
+  >
+    <BaseScroll class="x-list__item--expand">
+      <Sort class="x-list--padding-04 x-list--padding-bottom" />
```
Comment (ajperezbau) [inline src/components/mobile/mobile-aside.vue:11]: These classes can be deleted if the facets headers spacings are set using the padding token.
Also the CustomFacets ones.

File: src/components/mobile/mobile-close-aside.vue [hunk 7]
```diff
@@ -1,16 +1,43 @@
 <template>
-  <BaseIdModalClose modalId="aside-modal" class="x-button x-button--pill">
-    <FiltersIcon />
-    <span>{{ $t('toggleAside.hideAside') }}</span>
-  </BaseIdModalClose>
+  <div
+    class="
+      x-list x-list--horizontal x-list--gap-05
+      x-border-color--neutral-10
+      x-padding--05 x-padding--top-03 x-padding--bottom-04
```
Comment (ajperezbau) [inline src/components/mobile/mobile-close-aside.vue:6]:
```suggestion
      x-padding--00 x-padding--top-03 x-padding--bottom-04
```

**Review by ajperezbau — CHANGES_REQUESTED** [review]

**Comment by mnavarroespinosa** [comment]
The selected sorting option in the facet title font size should be 14px instead of 16px.

image

**Comment by mnavarroespinosa** [comment]
The space between the modal title and the facets list is too large:

image


Design:

image

**Review by tiborux — APPROVED** [review]

**Pull request merged**